tests/session/transaction-integration-002.phpt

```diff
@@ -64,6 +64,9 @@ try {
     echo $e->hasErrorLabel('TransientTransactionError') ?
         "found a TransientTransactionError" : "did NOT get a TransientTransactionError", "\n";
 }
+
+$sessionA->abortTransaction();
+$sessionB->abortTransaction();
```

Comment:
See: https://github.com/mongodb/mongo-php-library/pull/564#discussion_r203069307

After some more testing, it looks like this isn't actually necessary. In [endSession changes](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#endsession-changes) the spec requires that an in-progress transaction be aborted when the session ends. This may only be a problem in https://github.com/mongodb/mongo-php-library/pull/564#discussion_r202395078 because of a libmongoc bug.

Since neither of us could reproduce the blocking I saw in PHPLIB in these PHPC tests, I'd suggest removing these explicit aborts. Even if libmongoc has a bug, it looks like it does abort the transaction before the process terminates.